config.yaml

```diff
@@ -1,5 +1,39 @@
 ---
 Standards:
+  -
+    Title: Industry Prep Fridays
+    Description: "Activities and Lessons for Industry Prep Fridays"
+    UID: 2f1f90e3-3a7e-4647-bc39-47863a981d0e
+    SuccessCriteria:
+      - success criteria
+    ContentFiles:
+      -
+        Type: Instructor
+        UID: 38431de4-5a77-4820-8c9c-c30877c3d892
+        Path: /industry-prep-fridays/industry-prep-fridays.instructor.md
+      -
+        Type: Lesson
+        UID: defb503c-4b0e-4cbc-b524-8b912c4ebe35
+        Path: /industry-prep-fridays/intro-industry-prep-fridays.md
+      -
+        Type: Lesson
+        UID: b9f55320-922f-4ded-8240-0c7953fe2de8
+        Path: /industry-prep-fridays/asking-questions.md
+      -
+        Type: Lesson
+        UID: 47b86d79-b723-4cad-97a8-49906e78ce31
+        Path: /industry-prep-fridays/reading-documentation.md
```

Comment:
I think we made a change to this in the C17 Calendar Planning spreadsheet. The sequence is:

- Intro to Industry Prep
- Git Activity
- Code Coverage
- Asking Questions / Reading Documentation

There was a reason behind merging asking questions and reading documentation together, but I can't remember now. I think Code Coverage needed to be introduced earlier than we thought.

Reply:
Yep, we wanted to cover Code Coverage earlier. Thanks!